### PR TITLE
update config:satellite-vm

### PR DESCRIPTION
**Pull request by miteshrh**
SUMMARY

Config: satellite-vm has extended to support multi cloud like aws and osp

ISSUE TYPE
- Feature enhanced

### CODE REVIEW

File: ansible/configs/satellite-vm/default_vars_ocp.yml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+use_dynamic_dns: true
+osp_project_create: true
+student_name: student
+admin_user: opentlc-mgr
+
+
+bastion_instance_type: 2c2g30d
+satellite_instance_type: 8c32g100d
+
+___image: rhel-server-7.7-update-2
```
Comment (stencell) [inline ansible/configs/satellite-vm/default_vars_ocp.yml]: why so many _____ for this var? Makes it hard to determine if there is a mistake at a glance.

File: ansible/configs/satellite-vm/default_vars.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+env_type: satellite-vm 
+output_dir: /tmp/workdir                # Writable working scratch directory
+email: "{{env_type}}@example.com"
+guid: defaultguid
+
+# for file based repo
+repo_method: file 
+use_own_repos: true
+repo_version: "6.4"
```
Comment (stencell) [inline ansible/configs/satellite-vm/default_vars.yml]: Why such an old repo? Any reason not to update so you get updated packages with security and other fixes?

File: ansible/configs/satellite-vm/default_vars_ec2.yml [hunk 3]
```diff
@@ -1,17 +1,14 @@
...
+HostedZoneId: Z3IHLWJZOU9SRT
+aws_region: ap-southeast-2
 
-default_key_name: ~/.ssh/{{key_name}}.pem
-tower_run: false
 
 # How many do you want for each instance type
-
-
 bastion_instance_type: "t2.medium"
```
Comment (stencell) [inline ansible/configs/satellite-vm/default_vars_ec2.yml]: Consider moving to t3 for more future proof if you are already making and testing changes.

File: ansible/configs/satellite-vm/default_vars_ec2.yml [hunk 4]
```diff
@@ -1,17 +1,14 @@
...
 bastion_instance_image: RHEL75
```
Comment (stencell) [inline ansible/configs/satellite-vm/default_vars_ec2.yml]: Why such an old version of RHEL?

File: ansible/configs/satellite-vm/default_vars.yml [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+common_packages:
+  - python
+  - unzip
+  - bash-completion
+  - tmux
+  - wget
+  - git
+  - vim-enhanced
+  - at
+  - python27-python-pip
```
Comment (stencell) [inline ansible/configs/satellite-vm/default_vars.yml]: Any chance you could look at moving to python3 with these changes to be more future-proof? Several of the common roles have already been adapted to be compatible with either and we are exclusively python3 with the latest OCP on OSP config.

File: ansible/configs/satellite-vm/default_vars_ocp.yml (hunk 1 above)
Comment (miteshrh) [reply ansible/configs/satellite-vm/default_vars_ocp.yml]: changed to one _  thanks.

File: ansible/configs/satellite-vm/default_vars_ec2.yml (hunk 3 above)
Comment (miteshrh) [reply ansible/configs/satellite-vm/default_vars_ec2.yml]: Changed to t3.medium.

File: ansible/configs/satellite-vm/default_vars.yml (hunk 2 above)
Comment (miteshrh) [reply ansible/configs/satellite-vm/default_vars.yml]: Yeah, This is repo_version=6.4 is for satellite v6.4 and as of now we are doing only satellite 6.4. Hence no need to update, we need to keep it as is.

File: ansible/configs/satellite-vm/default_vars_ec2.yml (hunk 4 above)
Comment (miteshrh) [reply ansible/configs/satellite-vm/default_vars_ec2.yml]: Satellite 6.4 supports only rhel 7 hence at time of writing config I used RHEL75, Now if I want to change then it requires more time to test bit by bit of satellite installation and configuration. So I will keep this version, may be later we will change.

File: ansible/configs/satellite-vm/default_vars.yml (hunk 5 above)
Comment (miteshrh) [reply ansible/configs/satellite-vm/default_vars.yml]: Few of our modules which we use daily does not like python3 i.e. os_auth is still using python2.7 based openstacksdk so we like to keep as it is.

Comment (stencell) [reply ansible/configs/satellite-vm/default_vars.yml]: Your call...we use python3 for ocp4-disco environment and that includes openstacksdk as well.

**Comment by miteshrh** [comment]
@stencell I have made changes as asked and made some changes as per suggestion also. Please review and merge.

**Pull request merged**